pubspec.yaml

```diff
@@ -88,10 +88,9 @@ dependencies:
 
   # to do: We will change it when the PR (https://github.com/dietfriends/dns_client/pull/9) in upstream repository will be merged
   dns_client:
-    path: ../dns_client
-    # git:
-      # url: https://github.com/aryan-more/dns_client.git
-      # ref: twake-supported
+    git:
+      url: https://github.com/aryan-more/dns_client.git
```

Comment:
can you make the PR to https://github.com/linagora/dns_client. Thanks and BRs

Reply:
Done

Reply:
merged, please change this

Reply:
@hoangdat merged